### DISCUSSION TITLE
New command Idea: bun update-interactive

### DISCUSSION THREAD

**Discussion by jnatmorris** [oven-sh/bun#7228] [Ideas]
The idea for a new command would basically copy the yarn command, `yarn upgrade-interactive`. 

For those who may not know this command allows for an interactive command line method for updating packageswith further information. Such as the differences in versions (installed vs newest), and subsequently the type of change (x.0.0, or a 0.x.0, or 0.0.x).

This addition to bun would give greater controller over managing packages, and their versions, and would be useful in larger projects where the number of dependencies are great.

**Comment by Eveeifyeve** [comment]
dupe of https://github.com/oven-sh/bun/issues/4895.
